src/SFA.DAS.AssessorService.Database/PostDeploymentScripts/QF-1912-AddIndexToContacts-GovUkIdentifier.sql.sql

```diff
@@ -0,0 +1,4 @@
+﻿/*
+	Add an index to Contacts.GovUkIdentifier column
+*/
+CREATE INDEX [IX_Contacts_GovUkIdentifier] ON [Contacts] (GovUkIdentifier)
```

Comment:
This should be added to the [Contacts] table definition not a post deployment script - this will try and create an index each time it is run - first time would be okay, second time will fail - by adding to the [Contacts] table definition the DACPAC will sync it correctly.

Reply:
Moved index to table definition